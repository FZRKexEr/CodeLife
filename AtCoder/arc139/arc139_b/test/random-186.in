100
62 9 20 7 19 10
4 15 12 16 1 15
24 5 16 15 4 16
18 16 16 13 3 15
13 16 9 12 3 17
81 8 6 14 18 20
78 2 3 6 18 19
23 6 14 8 10 15
95 8 8 1 13 5
24 3 4 14 6 19
71 17 4 9 8 16
46 7 15 11 12 1
29 12 5 7 3 15
39 18 16 9 14 12
41 6 3 16 7 19
24 9 5 20 16 14
22 4 15 7 11 20
62 14 18 4 10 15
38 4 8 20 18 9
66 15 18 3 16 20
10 17 4 9 14 5
13 8 18 10 16 1
24 4 18 13 20 8
42 17 6 19 7 16
38 8 15 16 14 3
72 5 2 1 16 19
47 7 11 6 9 17
7 3 18 7 4 6
88 20 19 8 8 19
47 19 19 12 19 15
57 16 11 3 13 10
100 1 10 6 2 10
29 1 3 11 2 19
8 7 7 20 14 14
14 7 20 16 2 13
96 19 2 5 10 15
1 10 5 14 15 17
15 8 13 17 19 8
69 19 20 20 3 13
9 4 1 19 14 14
41 19 3 11 16 16
20 15 20 11 12 1
88 8 16 8 16 12
98 13 7 3 2 12
95 16 15 1 10 10
42 10 16 12 14 16
14 18 1 6 17 13
73 8 13 4 19 16
3 13 11 16 9 15
37 4 17 9 17 14
94 13 16 12 2 8
47 4 18 18 2 17
69 10 18 15 2 19
1 7 12 5 12 4
62 18 7 18 12 4
65 18 6 18 19 19
82 12 2 9 3 19
96 19 1 15 17 6
2 2 15 8 3 7
27 2 8 20 3 18
91 4 19 2 2 9
16 1 1 20 2 9
10 19 2 10 3 15
99 20 18 4 2 17
98 13 15 15 11 11
86 10 10 3 4 4
13 10 19 7 6 5
39 4 2 6 6 3
71 12 14 7 1 7
87 4 5 1 11 1
20 4 8 18 1 6
65 16 8 11 10 13
68 13 10 15 9 9
12 8 1 9 2 15
4 8 16 11 6 3
9 12 7 5 16 11
51 4 10 2 5 9
34 18 1 13 8 5
51 17 12 1 5 13
41 16 5 19 20 12
10 18 18 1 3 2
66 18 18 3 13 13
19 15 12 7 14 16
7 14 16 13 14 2
29 9 12 6 10 9
34 3 13 19 3 7
55 7 20 3 19 4
79 6 12 5 14 12
49 6 1 2 16 2
57 3 18 10 5 11
87 11 19 9 6 20
60 14 12 15 16 14
15 18 4 6 7 10
42 10 13 13 17 2
99 1 7 16 17 15
39 9 1 2 7 4
49 4 16 12 1 13
55 6 20 2 19 17
84 3 1 2 12 16
25 7 6 16 19 12
