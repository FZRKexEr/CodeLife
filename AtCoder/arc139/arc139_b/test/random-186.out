44
64
65
29
65
194
468
49
62
48
59
14
21
55
93
58
52
53
94
72
28
66
54
68
20
72
68
15
96
138
55
100
58
34
4
55
14
42
219
42
48
1
76
26
60
46
84
99
48
55
20
49
44
5
58
166
108
91
3
59
45
32
75
29
92
50
27
63
42
19
5
51
84
38
44
21
47
88
15
80
10
75
42
91
42
28
49
91
98
38
74
70
48
45
226
34
24
64
168
55
